packages/components/src/dropdown/index.js

```diff
@@ -15,6 +15,12 @@ class Dropdown extends Component {
 		this.close = this.close.bind( this );
 		this.clickOutside = this.clickOutside.bind( this );
 		this.bindContainer = this.bindContainer.bind( this );
+		/**
+		 * Add a ref to the Popover so that when contents change height due to user
+		 * interaction, the Popover instance can be accessed, including its class
+		 * function `refresh`.
+		 */
+		this.popoverRef = createRef();
```

Comment:
So if I understand properly, you're going to call `dropdown.popoverRef.refresh()` right? Would it make sense to add a `refresh` function to the `Dropdown` component instead?